cc/compile_and_test_inner.sh

```diff
@@ -27,6 +27,7 @@ function print_compile_options() {
     echo "enable_protocol_mpc_securenn: ${rtt_enable_protocol_mpc_securenn}"
     echo "   enable_protocol_mpc_helix: ${rtt_enable_protocol_mpc_helix}"
     echo "               enable_128bit: ${rtt_enable_128bit}"
+    echo "      enable_shape_inference: ${rtt_enable_shape_inference}"
```

Comment:
how about making it more adaptive and intelligent? Such that if the current python environment is '3.7', this option is default enabled.

Reply:
just add some shell code to check the environment and set the value according to the checking result.